interfaces/portal/src/app/pages/program-registration-debit-cards/components/link-card-on-site-dialog/link-card-dialog.component.html

```diff
@@ -58,84 +59,53 @@ <h3 i18n>{{ dialogLabels().header }}</h3>
           showTokenCodeInvalidWarning() && 'Fill up all 19 digits to link card'
         "
       />
-      <div class="w-full">
-        <div class="flex justify-end gap-2">
-          <p-button
-            i18n
-            (click)="closeDialog.emit()"
-            rounded
-            outlined
-            label="Cancel"
-            i18n-label
-            class="mt-4"
-          ></p-button>
-          <p-button
-            i18n
-            (click)="linkCard()"
-            rounded
-            [label]="dialogLabels().confirmationButton"
-            i18n-label
-            class="mt-4"
-          ></p-button>
-        </div>
-      </div>
     </div>
-  }
-  @if (linkCardDialogState() === linkCardDialogStates.errorAlreadyLinked) {
+  } @else {
     <ng-template pTemplate="header">
       <div class="flex items-center gap-2">
         <i class="pi pi-exclamation-triangle"></i>
         <h3 i18n>Error - Link visa card</h3>
       </div>
     </ng-template>
     <div class="flex flex-col gap-4">
-      <p i18n>
-        The card number you entered is already linked to another registration.
+      <p>
+        {{ errorMessage() }}
       </p>
-      <p i18n>
-        Please link this registration to a different card and inform your
-        supervisor.
+      <p>
+        {{ errorInstructions() }}
       </p>
-      <div class="w-full">
-        <div class="flex justify-end">
-          <p-button
-            i18n
-            (click)="closeDialog.emit()"
-            rounded
-            label="Go back"
-            i18n-label
-          ></p-button>
-        </div>
-      </div>
     </div>
-  }
-  @if (linkCardDialogState() === linkCardDialogStates.errorNotFound) {
-    <ng-template pTemplate="header">
-      <div class="flex items-center gap-2">
-        <i class="pi pi-exclamation-triangle"></i>
-        <h3 i18n>Error - Link visa card</h3>
-      </div>
-    </ng-template>
-    <div class="flex flex-col gap-4">
-      <p i18n>
-        Card number not found. Please go back and check that the number is
-        correct.
-      </p>
-      <p i18n>
-        If the number is correct but this error persists, try a different card
-        and inform your supervisor.
-      </p>
-      <div class="w-full">
-        <div class="flex justify-end">
-          <p-button
-            i18n
-            (click)="closeDialog.emit()"
-            rounded
-            label="Go back"
-            i18n-label
-          ></p-button>
-        </div>
-      </div>
+    <div class="w-full">
+      <div class="flex justify-end"></div>
     </div>
   }
+
+  <div class="w-full">
+    <div class="mt-4 flex justify-end gap-2">
+      <p-button
+        i18n-label="@@generic-cancel"
+        (click)="dialogVisible.set(false)"
+        rounded
+        outlined
+        label="Cancel"
+        i18n-label
```

Comment:
```suggestion
        (click)="dialogVisible.set(false)"
        rounded
        outlined
        label="Cancel"
        i18n-label="@@generic-cancel"
```

This should have been handled by some auto-fix lint-rule... But apparently that is not the case. Sad.